commitlint.config.js

```diff
@@ -1 +0,0 @@
-module.exports = { extends: ['awesome'] }
```

Comment:
Keep the config in this file, unless there's a specific reason to change it.

Reply:
I forgot to tell you why did I change this config file, I could tell you why in "Is there anything you'd like reviewers to focus on?" section of the PR.

Anyway here is the reason : I like consistency, and currently we have `.eslintrc.json`, so it feels better to do the same thing for this config file. If possible, we should try to be consistent in the codebase or for config files.

Reply:
Perhaps. But this is not the PR for that change.